jest-preset.json

```diff
@@ -24,5 +24,9 @@
   },
   "transformIgnorePatterns": [
     "node_modules/(?!@ngrx)"
+  ],
+  "snapshotSerializers": [
+    "<rootDir>/node_modules/jest-preset-angular/AngularSnapshotSerializer.js",
+    "<rootDir>/node_modules/jest-preset-angular/HTMLCommentSerializer.js"
```

Comment:
Shouldn't the order be different? Have you tested if that changes anything?

Reply:
Just tried both and seems there is no any difference. Will look how jest applies this config.

Reply:
I bet I've seen something about the order in jest docs, but can't find it now. Anyway, jest reverses the list:
```
config.snapshotSerializers
    .concat()
    .reverse()
    .forEach(path => {
      addSerializer(localRequire(path));
    });
```
So it's same as before.